### DISCUSSION TITLE
关于文件中时间的表达

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#100] [Q&A] [answered]
请问时间应该以年月日计还是要精确到秒？
![image](https://user-images.githubusercontent.com/100075314/156750927-7733961c-2e2d-409e-a311-09997ecb2708.png)
![image](https://user-images.githubusercontent.com/100075314/156751005-18a657bc-28a7-44f2-9d01-5643547b0456.png)

**Comment by binbin00** [comment, marked as answer]
除了指令日志中的时间，年月日就可以了。
